phpBB/phpbb/storage/adapter/local.php

```diff
@@ -331,4 +337,12 @@ public function file_image_height($path)
 	{
 		return $this->image_dimensions($path);
 	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function get_link($path)
+	{
+		return generate_board_url() . $this->path . $path;
```

Comment:
Please use more distinctive names. Now it seems like your adding a path twice. The class' path looks to be the storage adapter's base path and the `$path` seems to be the file path relative to this folder but your variable naming is not clear enough for my taste.

Reply:
Yes i know, is something i have to do in the future, but for me is the same name for different things

Reply:
I change it a little on split uploaded files into subdirectories